helpers/if.js

```diff
@@ -3,8 +3,8 @@
 const _ = require('lodash');
 
 const factory = globals => {
-    return function(lvalue, operator, rvalue) {
-        const options = arguments[arguments.length - 1];
+    return function(lvalue, operator, rvalue, options) {
+        options = options ? options : (rvalue ? rvalue : operator);
```

Comment:
It's not immediately apparent how this relates to the parameter shuffling that takes place below and why we wouldn't do this at the same time.

Reply:
added a comment